.gitignore

```diff
@@ -1,3 +1,2 @@
 .byebug_history
-Gemfile.lock
```

Comment:
This is deliberately not added to the repo.
Check out https://yehudakatz.com/2010/12/16/clarifying-the-roles-of-the-gemspec-and-gemfile/